Assignment4/Assignment4.js

```diff
@@ -0,0 +1,24 @@
+var fs = require('fs');
+console.log("------------------------------------------");
+function ReadAppendWrite (read_file, append_file, write_file) {
```

Comment:
- variable or parameter names must be _camelCased_ , not _underscore_separated_
- function name not correctly _camelCased_